pkg/ccl/backupccl/backup_job.go

```diff
@@ -1156,9 +1156,7 @@ func (b *backupResumer) maybeNotifyScheduledJobCompletion(
 		}
 
 		scheduleID := int64(tree.MustBeDInt(datums[0]))
-		if err := jobs.NotifyJobTermination(
-			ctx, txn, env, b.job.ID(), jobStatus, b.job.Details(), scheduleID,
-		); err != nil {
+		if err := jobs.NotifyJobTermination(ctx, txn, env, b.job.ID(), jobStatus, jobErr, b.job.Details(), scheduleID); err != nil {
```

Comment:
instead of refactoring the `NotifyJobTermination` interface, why not increment the metric here, before we call `NotifyJobTermination`?

Reply:
I think in this case we'd need to pass in the same bool that tells the job whether or not it should increment the KMS into the backup job. But I also think that it's not a bad idea to have `jobs.NotifyJobTermination` to notify with the error either since there may be other use cases for this in the future.

Reply:
I moved the metric to the BACKUP statement instead. PTAL